spec/features/form/check_answers_page_lettings_logs_spec.rb

```diff
@@ -167,14 +167,12 @@
       it "displays inferred postcode with the location id" do
         lettings_log.update!(location:)
         visit("/lettings-logs/#{id}/setup/check-answers")
-        expect(page).to have_content("Location")
         expect(page).to have_content(location.name)
       end
 
       it "displays inferred postcode with the location_admin_district" do
         lettings_log.update!(location:)
         visit("/lettings-logs/#{id}/setup/check-answers")
-        expect(page).to have_content("Location")
         expect(page).to have_content(location.location_admin_district)
       end
     end
```

Comment:
I wasn't able to update this test to 2023 easily without major changes to the rest of the tests. Just deleted the test for the check answer label. As it doesn't appear anymore pre 2023.